get_prism_data.R

```diff
@@ -54,6 +54,7 @@ download_prism=function(){
 #########################################################
 check_if_prism_files_present=function(prism_ls, years){
   #Extract out all the variable names and dates
+  if (length(prism_ls$files) == 0){return(FALSE)}
```

Comment:
This seems like [documenting mechanics instead of design and purpose](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745#s8). If you don't what is happening is clear from the way the code is written I'd be happy to rewrite the code. E.g., we could use an if/else here instead of the if/return if that would be clearer, or an an explicit `num_files` variable.